depends/patches/miniupnpc/dont_use_wingen.patch

```diff
@@ -0,0 +1,23 @@
+commit e8077044df239bcf0d9e9980b0e1afb9f1f5c446
+Author: fanquake <fanquake@gmail.com>
+Date:   Tue Aug 18 20:50:19 2020 +0800
+
+    Don't use wingenminiupnpcstrings when generating miniupnpcstrings.h
+
+    The wingenminiupnpcstrings tool is used on Windows to generate version
+    information. This information is irrelevant for us, and trying to use
+    wingenminiupnpcstrings would cause builds to fail, so just don't use it.
```

Comment:
Is it worth to mention that this patch could be skipped on miniupnpc versions 2.1+ ?
See: https://github.com/miniupnp/miniupnp/commit/9663c55c61408fdcc39a82987d2243f816b22932

Reply:
Will be added in the next push.